lib/rules/no-top-level-side-effects.ts

```diff
@@ -141,6 +145,9 @@ export const noTopLevelSideEffects: Rule.RuleModule = {
       allowedNews: provided?.allowedNews || allowedNewsOption.default,
       allowIIFE: provided?.allowIIFE || allowIIFEOption.default,
       allowDerived: provided?.allowDerived || allowDerivedOption.default,
+      allowFunctionProperties:
```

Comment:
This should be added to the [`schema`](https://github.com/ericcornelissen/eslint-plugin-top/blob/e760ffab5f777ec26665f8f47daba6cc6eeafc2b/lib/rules/no-top-level-side-effects.ts#L97-L130) too.

Reply:
Resolved with cfd4465780362ebfe56af734be9e6abe549067de